src/quo2/components/buttons/button/type_values.cljs

```diff
@@ -0,0 +1,186 @@
+(ns quo2.components.buttons.button.type-values
```

Comment:
Forgot to mention in my previous review, but I'm not in love with `type-values` as the name of the ns 🤷🏼

Reply:
haha me neither, I will make something better